kore/src/Kore/Builtin/Set.hs

```diff
@@ -686,17 +677,17 @@ unifyEquals
     unifyEqualsElement
         :: Domain.InternalSet  -- ^ concrete set
         -> SymbolOrAlias  -- ^ 'element' symbol
-        -> p  -- ^ key
-        -> unifier (expanded, proof)
+        -> TermLike variable  -- ^ key
+        -> unifier (Pattern variable)
     unifyEqualsElement builtin1 element' key2 =
         case Set.toList set1 of
             [fromConcreteStepPattern -> key1] ->
                 do
-                    (key, _) <- unifyEqualsChildren key1 key2
+                    key <- unifyEqualsChildren key1 key2
                     let result =
```

Comment:
Unnecessary binding. (These will all get caught and removed gradually, I assume, which is the way It Should Be.)